Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.yml

```diff
@@ -62,6 +99,48 @@ script:
       - contextPath: AnomaliSecurityAnalytics.SearchJob.job_id
         description: Job ID of the search job.
         type: String
+  - name: anomali-security-analytics-search-job-status
+    description: Get the status of one or more search jobs.
+    arguments:
+    - name: job_id
+      description: Unique identifier assigned to a background process or job.
```

Comment:
As this can get a ids in CSV, please update the description about this, and update also the README
for example, see `splunk-job-status` command in Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml